File: 017-Palindrome Number Finder/PalindromeNumberFinder.py
def is_palindrome(n):
    return str(n) == str(n)[::-1]

# Input format: e.g., 10, 80
input_str = input()
lower, upper = map(int, input_str.split(','))

palindromes = []

for i in range(lower, upper + 1):
    if is_palindrome(i):
        palindromes.append(str(i))

# Join and print with commas, ending with a period
print(", ".join(palindromes) + ".")

File: 017-Palindrome Number Finder/PalindromeNumberFinder.cpp
#include<iostream>
using namespace std;

bool isPalindrome(int num){
    int original = num, reversed = 0;
    while(num > 0){
        int digit = num % 10;
        reversed = reversed * 10 + digit;
        num /= 10;
    }
    return original == reversed;
}

int main(){
    int lower, upper;
    char comma;
    cout << "Enter the range (lower, upper): ";
    cin >> lower >> comma >> upper;
    cout << "Palindrome numbers in the range [" << lower << ", " << upper << "] are: ";
    for(int i = lower; i <= upper;i++){
        if(isPalindrome(i)){
            cout<< i << " ";
        }
    }
    cout << endl;
    return 0;
}